localization/autoware_pose_initializer/src/pose_initializer_core.hpp

```diff
@@ -15,15 +15,17 @@
 #ifndef POSE_INITIALIZER_CORE_HPP_
 #define POSE_INITIALIZER_CORE_HPP_
 
-#include <autoware/component_interface_specs_universe/localization.hpp>
-#include <autoware/component_interface_utils/rclcpp.hpp>
+#include <autoware/component_interface_specs/localization.hpp>
 #include <autoware_utils_diagnostics/diagnostics_interface.hpp>
 #include <autoware_utils_logging/logger_level_configure.hpp>
 #include <rclcpp/rclcpp.hpp>
 
+#include <autoware_adapi_v1_msgs/msg/localization_initialization_state.hpp>
+#include <autoware_internal_localization_msgs/srv/initialize_localization.hpp>
```

Comment:
Since the dependency on autoware_internal_localization_msgs has been removed from package.xml, should this header also be removed?